api/helpers/email.js

```diff
@@ -146,25 +146,34 @@ exports.sendWelcomeEmail = async function (projectName, email) {
  * @returns {void}
  */
 const sendEmail = async (emailTemplate) => {
-    const emailToken = await getEmailToken();
+    try {
+        // Retrieve and validate token
+        const emailToken = await getEmailToken();
+        const token = emailToken?.data?.access_token;
+        if (!token) {
+            defaultLog.error("Couldn't get a valid CHES token", emailToken);
+            return;
+        }
 
-    if (emailToken && emailToken.data && emailToken.data.access_token) {
         // Send confirmation email to user.
-        await axios.post(
+        const response = await axios.post(
             _commonHostingEmailServiceEndpoint + _CHES_emailMergeAPI,
             emailTemplate,
             {
                 headers: {
-                    "Authorization": 'Bearer ' + emailToken.data.access_token,
+                    "Authorization": `Bearer ${token}`,
                     "Content-Type": 'application/json'
                 }
             }
-        
-        )
-        .then((response) => defaultLog.info("Email Sent:", response.data))
-        .catch((error) => defaultLog.info('Email not sent: ', error.response.data))
-    } else {
-        defaultLog.error("Couldn't get a proper token", emailToken);
+        );
+
+        defaultLog.info('Email sent:', response.data);
+    } catch (error) {
+        if (error.response?.data) {
+            defaultLog.error('Ches rejected the email:', error.response.data);
```

Comment:
```suggestion
            defaultLog.error('Ches rejected the email:', error);
```

I think it would be nice to see the full error object here, it may include other bits of helpful troubleshooting info

Reply:
Changed to full object